packages/api/test/functional/user-bookmark.spec.js

```diff
@@ -204,17 +225,26 @@ test('GET /user_bookmarks admin user gets all users that bookmarks a specific te
 	const loggeduser = await User.create(adminUser);
 	const user1 = await User.create(user);
 	const user2 = await User.create(otherUser);
-	const technologyIds = await Technology.ids();
-	await user1.bookmarks().attach(technologyIds);
-	await user2.bookmarks().attach(technologyIds);
+	const tech1 = await Technology.create(technology);
+	await user1.bookmarks().attach([tech1.id]);
+	await user2.bookmarks().attach([tech1.id]);
 
 	const response = await client
-		.get(`user_bookmarks?technologyId=${technologyIds[0]}`)
+		.get(`user_bookmarks?technologyId=${tech1.id}`)
 		.loginVia(loggeduser, 'jwt')
 		.end();
 
+	const result = await User.query()
+		.whereHas('bookmarks', (builder) => {
+			builder.where({ technology_id: tech1.id });
+		})
+		.with('bookmarks', (builder) => {
+			builder.where({ technology_id: tech1.id });
+		})
```

Comment:
```suggestion
		.with('bookmarks')
		.whereHas('bookmarks', (builder) => {
			builder.where({ technology_id: tech1.id });
		});
```

Seguindo meu comentário lá em cima, não testei mas suspeito que da forma acima funcione.